dataland-frontend/src/components/general/DataPointDataTableInModal.vue

```diff
@@ -0,0 +1,90 @@
+<template>
```

Comment:
![image](https://github.com/d-fine/Dataland/assets/104447806/11f47e82-ed3b-4d55-90e3-bad31c7eb318)
Can the auto-focused on the "x" removed?

Reply:
done

Reply:
Thank you